query-service/src/integrationTest/resources/attribute-expression-test-queries/query2.json

```diff
@@ -0,0 +1,85 @@
+{
+  "filter": {
+    "childFilter": [{
+      "lhs": {
+        "attributeExpression": {
+          "attributeId": "EVENT.startTime"
+        }
+      },
+      "operator": "GT",
+      "rhs": {
+        "literal": {
+          "value": {
+            "valueType": "LONG",
+            "long": "1570658506605"
+          }
+        }
+      }
+    }, {
+      "lhs": {
+        "attributeExpression": {
+          "attributeId": "EVENT.startTime"
+        }
+      },
+      "operator": "LT",
+      "rhs": {
+        "literal": {
+          "value": {
+            "valueType": "LONG",
+            "long": "2570744906673"
+          }
+        }
+      }
+    }, {
+      "lhs": {
+        "attributeExpression": {
+          "attributeId": "EVENT.spanTags"
+        }
+      },
+      "operator": "CONTAINS_KEY",
+      "rhs": {
+        "literal": {
+          "value": {
+            "string": "span.kind"
+          }
+        }
+      }
+    }, {
+      "lhs": {
+        "attributeExpression": {
+          "attributeId": "EVENT.spanTags",
+          "subpath": "span.kind"
+        }
+      },
+      "operator": "NEQ",
+      "rhs": {
+        "literal": {
+          "value": {
+          }
+        }
+      }
+    }]
+  },
+  "selection": [{
+    "function": {
+      "functionName": "AVG",
+      "arguments": [{
+        "attributeExpression": {
+          "attributeId": "EVENT.duration"
+        }
+      }],
+      "alias": "avg_duration"
+    }
+  }, {
+    "attributeExpression": {
```

Comment:
Can we also add aggregation function on map attribute expression with subpath - e.g `sum(spanTags__Keys, 'span.kind', spanTags__values) ?

Reply:
Please add these tests in followup

Reply:
Does this sample query work for this case ?
```
select sum(mapValue(tags__KEYS, 'otel.status_code', tags__VALUES)) from spanEventView
```

Reply:
added